angular-client/src/components/battery/battery.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="battery">
+  >
+  <div
+    class="battery-nub"
+    (click)="toggleIsCorrect()"
+    [@toggleClick]="clicked"
+    [ngStyle]="{
+      height: nubHeight,
+      width: nubWidth,
+      'background-color': backColor,
+      'border-top-left-radius': roundCorner,
+      'border-top-right-radius': roundCorner
+    }"
+  ></div>
+  <div
+    class="battery-bar"
+    [ngStyle]="{
+      'background-color': fillColor,
+      height: heightpx,
+      width: widthpx,
+      'border-radius': roundCorner
+    }"
+  >
+    <ng-template [ngIf]="full == false">
```

Comment:
this works when at 100 but fails for values like 95 and above pretty much when the height and width are 100, you have to figure out a way to calculate which value is the full threshold

Reply:
spent way too long trying to fix this and eventually got something that works. i need to not work on this battery anymore pls LOL